core/lib/globals_shared.h

```diff
@@ -1838,6 +1838,11 @@ typedef union _dr_zmm_t {
     reg_t reg[IF_X64_ELSE(8, 16)]; /**< Representation as 8 or 16 registers. */
 } dr_zmm_t;
 
+/* OpMask (k-)register. The register may be only 16 bits on systems w/o AVX512BW, but
```

Comment:
If this type will be public in the API (which it must be since this PR lists in release.dox, though I don't see it used in any struct here?), this needs to be a Doxygen comment and more formally worded for public-facing documentation: i.e., spell out "w/o" and probably add more context before "OpMask".

Reply:
I changed the public doxygen comment to 
"OpMask AVX-512 (k-)register."
That should be enough, as it's unambiguous, similar to "512-bit ZMM register.". I moved the rest into AVOID_API_EXPORT.

Reply:
(It's used in the test, and will be used e.g. in mcontext.)